### DISCUSSION TITLE
Headline (tag) in Headers

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1775] [Proposals]
### Goal

I would like to have a headline (tag) in headers. We need to continue the experience throughout the app.

### Context

I am creating the bill flow in My O2 and trying to make the same experience throughout the app. If users see an tag with the bill status in some screens, when they land on the bill details, they don't see the same visual representation of this information.

### Impact

Create an option in Headers to show a headline (tag)

### Discussion Questions

Could we have this option in headers?

### Do you have a visual proposal?

https://www.figma.com/design/qKsUe3xW8wDOoySLOzWw3I/My-O2-App-Master-2024?node-id=233-5713&t=mv3c5PBZEsmmDjmT-4

**Comment by HectorSancho** [comment]
We have been talking about it in Design Core, and finally we are going to include the possibility of including tags in the header.

In this case (and potentially in others), it makes sense in terms of UX and also for coherence with other DS components such as cards or heros.
